src/Geometry/NewestVertexBisection.jl

```diff
@@ -0,0 +1,308 @@
+# using Gridap.Arrays
+using SparseArrays
+using TimerOutputs
+using Random
+
+# Create a TmerOutput, this is the main type that keeps track of everything.
+const to = TimerOutput()
```

Comment:
We still need to decide which timing mechanism adopt for internal functions in Gridap.

For the moment, I would remove the timings until we have decided how to do this for other parts of Gridap. Hope it is not a big inconvenience for you.

Another temporary option is to remove the global constant `to` and pass it to the `newest_vertex_bisection` as an optional argument.